pkg/report/cyclonedx/cyclonedx.go

```diff
@@ -363,3 +418,142 @@ func property(key, value string) cdx.Property {
 		Value: value,
 	}
 }
+
+func advisories(refs []string) *[]cdx.Advisory {
+	var advs []cdx.Advisory
+	for _, ref := range refs {
+		advs = append(advs, cdx.Advisory{
+			URL: ref,
+		})
+	}
+	return &advs
+}
+
+func cwes(cweIDs []string) *[]int {
+	var ret []int
+	if len(cweIDs) == 0 {
+		return nil
+	}
```

Comment:
Because Vendor Advisory does not always return the CWE-IDs, I returned nil. 

Although not explicitly stated in the documentation 
probably, by returning an empty array, the absence of a CWE-ID is clearly expressed.

Reply:
I suggested as below.
https://go.dev/play/p/KFrouKaN4HV

Reply:
Thank you! Understood...